NOTICE

```diff
@@ -0,0 +1,7 @@
+kubesnap-plugin-collector-docker
```

Comment:
Please consider changing from: "kubesnap-plugin-collector-docker" to: "snap-plugin-collector-docker".